oci/private/pull.bzl

```diff
@@ -183,6 +183,7 @@ _SUPPORTED_MEDIA_TYPES = {
     "manifest": [
         "application/vnd.docker.distribution.manifest.v2+json",
         "application/vnd.oci.image.manifest.v1+json",
+        "application/vnd.cncf.helm.config.v1+json",
```

Comment:
can you come up with an example we could add to /WORKSPACE so there's a local reproduction we can use to test? I tried the literal helm chart you put in the PR description but get a 401 fetching that, I assume it's in a private registry.